### PR TITLE
Implement and test debit-txns

### PR DESCRIPTION
**Pull request by mhuerster**
cc @damien @devmeyster

### CODE REVIEW

File: src/user.clj [hunk 1]
```diff
@@ -57,5 +58,10 @@
    date :- TimeStamp]
   (filter #(same-week? (:date %1) date) txns)))
 
+(s/defn debit-txns :- [MapWithAmount]
+  "Filters transactions to only include debits."
+  [txns :- [MapWithAmount]]
+  (filterv #(< 0 (:amount %1)) txns))
```
Comment (mhuerster) [inline src/user.clj]: returns a vector rather than a list

**Comment by damien** [comment]
👍

**Comment by mhuerster** [comment]
fixed merge conflicts - please ♻️

**Comment by damien** [comment]
Do we still need this?

**Comment by mhuerster** [comment]
probably not

**Pull request closed without merging**